### ISSUE TITLE
`super` method not present in type declarations

### ISSUE THREAD

**Issue by jhpratt** [babel/babel#10075]
## Bug Report

**Current Behavior**
Calling `t.super()` results in a compile-time type error, as `super` isn't present in type definitions. It does exist in [the documentation](https://babeljs.io/docs/en/babel-types#super) and works as expected after compilation.

**Expected behavior**
The `super` method should be present in the type definitions.

**Babel Configuration (.babelrc, package.json, cli command)**
Not applicable.

**Environment**
- Babel version(s): v7.4.0
- Node/npm version: Node 12.0.0, npm 6.9.0
- OS: Ubuntu 19.04
- Monorepo: no
- How you are using Babel: CLI

As someone using TypeScript to minimize errors, the amount of casting is _extremely_ verbose. By itself, that's not an issue, but coupled with errors in type definitions (this along with #10065) are forcing the use of unsafe casts and `// @ts-ignore`.

**Comment by nicolo-ribaudo** [comment]
I'm not a TS expert, but how could we write this type definition so that it is valid code?
```ts
export function super() {}
```

**Comment by jhpratt** [comment]
@nicolo-ribaudo That should be sufficient. It's my understanding type definitions are auto-generated, though, so I'm not sure where the error is truly at.

**Comment by nicolo-ribaudo** [comment]
No, `super` is a keyword.

**Comment by tanhauhau** [comment]
I figured we can do
```
declare function _super(): Super;
export { _super as super }
```

**Comment by JLHwung** [comment]
Fixed by #10098
